pkg/limayaml/default.yaml

```diff
@@ -57,6 +57,10 @@ ssh:
   # Default: true
   loadDotSSHPubKeys: true
 
+# # extra environment variables that will be loaded into the VM at start up
+# environmentVariables:
```

Comment:
I would prefer the key to be just `env` here. I care less about how we call it inside the Go code.

Hidden agenda: I hope that one day we can support some kind of override mechanism on the commandline, so you can write:

```
limactl start examples/arch.yaml --set env.HTTP_PROXY=my.proxy.com:8080
```

(kind of like the way `helm` allows you to override `values.yaml` from the commandline). And for that I want the name to be short. 😄